spring-boot-starter-liquibase-couchbase-test/src/test/java/org/liquibase/ext/couchbase/starter/test/CreateCollectionSystemTest.java

```diff
@@ -0,0 +1,41 @@
+package org.liquibase.ext.couchbase.starter.test;
+
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.manager.collection.CollectionSpec;
+import com.couchbase.client.java.manager.collection.ScopeSpec;
+import org.junit.Test;
```

Comment:
I think you should use 'import org.junit.jupiter.api.Test' import instead of 'import org.junit.Test'

Reply:
++